src/main/resources/db/migration/V0.5.15__create-inventory-tables.sql

```diff
@@ -0,0 +1,193 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TYPE "entity_level" AS ENUM (
+  'POPULATION',
+  'INDIVIDUAL'
+);
+
+CREATE TYPE "entity_relationship_type" AS ENUM (
+  'POPULATION_OF'
+);
+
+CREATE TABLE entity (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    name text NOT NULL,
+    level entity_level,
+    cross_id UUID,
+    active boolean DEFAULT true,
+    status_date timestamptz(0),
+    status_by UUID,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE entity ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE entity ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+COMMENT ON COLUMN entity.id IS 'Used as externalReference to Brapi Germplasm';
+
+CREATE TABLE entity_relationship (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    type entity_relationship_type,
+    relation_source_id UUID,
+    relation_target_id UUID,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE entity_relationship ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE entity_relationship ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE entity_cross (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    mother_entity UUID NOT NULL,
+    father_entity UUID,
+    name text,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE entity_cross ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE entity_cross ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+COMMENT ON COLUMN entity_cross.id IS 'Used as externalReference to Brapi Cross';
+
+CREATE TABLE inventory (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    entity_id UUID NOT NULL,
+    amount integer,
+    units UUID NOT NULL,
```

Comment:
Can we do units_id for consistency?

Reply:
yep, I'll change that.

Reply:
Also added to mother_entity/father_entity in entity_cross table. Left status_by in entity table since other user references are made in that way in base_edit_track_entity. Pushed changes.